### ISSUE TITLE
[Release notes] Add known issue about Alerts table and threat indicator alerts

### ISSUE THREAD

**Issue by nastasha-solomon** [elastic/security-docs#3609]
## Description 

The following known issues will need to be documented in the 8.9 release notes, and possibly the 8.x release notes:

NOTE: These summaries are still under development. @nastasha-solomon is discussing them with @kqualters-elastic asynchronously. 

- An EUI bug can affect the height at which the Alerts table renders, which makes it difficult to view alerts.
- Alerts generated from threat indicator match rules won’t display in the Alerts table because of an issue with the nested `file.name` field

## Workarounds
There are multiple workarounds for these bugs, but they might depend on whether non-threat-indicator-match alerts are mixed in with threat indicator match alerts in the Alerts table.

**Comment by nastasha-solomon** [comment]
Release notes summary for bug # 2:

Rendering issues can occur within the Alerts table if threat indicator match alerts contain nested `file.name` values and the `file.name` column is included in the Alerts table. This issue stems from a known bug that occurs when the Alerts table sends a request to the Elasticsearch Fields API ([#97684](https://github.com/elastic/elasticsearch/issues/97684)).

Resolving this issue depends on the types of alerts displayed in the Alerts table. Choose the workaround that is most relevant to your case:

**Case # 1:** You’re viewing threat indicator match alerts with nested file.name fields and the Alerts table doesn’t render.
In this situation, you will need to manually update your local storage to re-render the table:

1… (insert steps described [here](https://github.com/elastic/kibana/issues/159143#issuecomment-1644267086))

**Case # 2:** Your’re viewing a mix of threat indicator match alerts with nested file.name fields and other types of alerts. The Alerts table renders, but only displays empty rows.
To resolve this issue, remove the file.name field via the Fields browser and refresh the Alerts page.

**Comment by cthulhusec** [comment]
@nastasha-solomon I don't know if it's worth mentioning but Bug 2 also applies to the `email.attachtments` nested field with the `file.name` field. I don't believe this is currently in use by any of y'alls products but would impact any customer making use of the published and available schema.

For instance, my org uses those fields for phishing email reports, so we are currently affected.


I can provide more info if you need it but wanted to make sure you knew about that other scenario incase you want to include it as well.

**Comment by kqualters-elastic** [comment]
@cthulhusec yes, there are possibly other "collisions" like this, where if a column name is selected via the fields browser and that same column name exists as a property of a nested field within any of the indices being queried, users will run into this unfortunately. Looks like it will happen for email.attachments.file.hash.* and file.hash.* as well
